src/main/java/io/appform/jsonrules/expressions/array/CollectionJsonPathBasedExpression.java

```diff
@@ -62,13 +63,8 @@ protected CollectionJsonPathBasedExpression(ExpressionType type, String path, @S
     @Override
     protected final boolean evaluate(ExpressionEvaluationContext context, String path, JsonNode evaluatedNode) {
         if (extractValues) {
-            JsonNode jsonNode = MissingNode.getInstance();
-            try {
-                jsonNode = mapper
-                        .valueToTree(JsonPath.read(context.getNode().toString(), String.valueOf(valuesPath)));
-            } catch (PathNotFoundException e) {
-                // consume silently; indicates path denoted by @valuesPath doesn't exist.
-            }
+            JsonNode jsonNode = mapper.valueToTree(JsonPath.using(ComparisonUtils.SUPPRESS_EXCEPTION_CONFIG)
+                    .parse(context.getNode().toString()).read(String.valueOf(valuesPath)));
             if (jsonNode == null || !jsonNode.isArray()) {
```

Comment:
No exactly relevant to this pull, but jsonNode == null .. check should also jave jsonNode.isNull() ?